tools/deb/test/run_pilot_test.sh

```diff
@@ -31,21 +31,19 @@ function build_all() {
 
 
   if [[ -d $GOPATH/src/istio.io/pilot ]]; then
-    (cd $GOPATH/src/istio.io/pilot; git pull upstream master)
+    (cd $GOPATH/src/istio.io/pilot; git fetch && git pull upstream master)
```

Comment:
why do you need to change pull to fetch & pull?
pull is just a short form of fetch + checkout. https://git-scm.com/docs/git-pull

Reply:
when I'm on master and/or my own branches I think I need to fetch for pull to do anything so I've had the habit of always doing "git fetch && git pull" but maybe the "upstream master" part does the fetch somehow/or doesn't make it apply

it can't hurt can it though ? (though if it's 100% sure it's never any different yes I shouldn't add a noop )

Reply:
looks like I must have been dreaming though so will revert those parts